test/README.md

```diff
@@ -109,3 +109,187 @@ For more options, see the help output for the `robot` command.
 ```
 $ ./_output/robotenv/bin/robot -h
 ```
+
+## Test Scenarios in CI
+
+The test scenario tools in the `bin` directory are useful for running
+more complex test cases that require VMs and different images.
+
+### Package Sources
+
+In order to build different images, Composer needs to be configured
+with all of the right package sources to pull the required RPMs. The
+sources used by all scenarios are in the `package-sources` directory.
+
+Package source definition files are templates using `envsubst`,
+which means the files can have shell variables embedded.
+
+```
+id = "fast-datapath"
+name = "Fast Datapath for RHEL 9"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel9/${UNAME_M}/fast-datapath/os"
+check_gpg = true
+check_ssl = false
+system = false
+rhsm = true
+```
+
+Refer to `./bin/build_images.sh` for the set of known variables that can
+be expanded.
+
+## Image Blueprints
+
+The image blueprints are independent of the test scenarios so that
+images can be reused. Be careful making changes to specific images in
+case the change affects the way the image is used in different
+scenarios. Be careful adding unnecessary new images, since each image
+takes time to build and may slow down the overall test job.
+
+Add blueprints as TOML files in the `image-blueprints` directory, then
+add a short description of the image here for reference.
+
+Name | Purpose
+---- | -------
+rhel-9.2 | A simple RHEL image without MicroShift.
+rhel-9.2-microshift-4.13 | A RHEL 9.2 image with the latest MicroShift 4.13 z-stream installed and enabled.
+rhel-92-microshift-source | A RHEL 9.2 image with the RPMs built from source.
+rhel-90-microshift-source | A RHEL 9.0 image with the RPMS built from source.
```

Comment:
I think the 9.2 vs 92 was a typo, I'll fix that.

Regarding the `/`, I was trying to be conservative about which characters are in the name because the same name is used as the blueprint name and ostree ref ID to make it easier for someone writing a test scenario to make sure they have the right value in place. I'm not sure if a blueprint can have a `/` in the name. If it can, and the scripting that uses the names won't be confused by the `/`, we could make that change. Can we do that in a follow-up? After we have all of this wired in and can automatically test a change it will be easier to ensure we have all of the changes we need.